Automattic/newspack-ios
# Issues/the great code purge

This PR removes obsolete code added during the v0.  As the v1 feature set has shifted we no longer need a lot of things we assumed we would when the v0 was developed. 

To test: 
Confirm the app builds. 
Run a quick smoke test and confirm there are no regressions.
Ensure unit tests pass.

@jleandroperez can I trouble you with this fun one? :D
